src/components/20-molecules/footer-small/index.js

```diff
@@ -0,0 +1,60 @@
+import { LitElement, html, css, unsafeCSS } from 'lit-element';
+import footerSmallCSS from './index.scss';
```

Comment:
We can omit file extensions `.scss`

Reply:
That only works in case of "js" files afaik. Still, I tried it out and it does not apply the scss then.